src/documentation/0002-node-history/index.md

```diff
@@ -39,8 +39,8 @@ Express is born
 ## 2011
 
 npm hits 1.0
-Big companies start adopting Node.js: LinkedIn, Uber
-[Hapi](https://hapijs.com) is born
+Big companies start adopting Node.js: LinkedIn, Uber,
```

Comment:
I don't believe there should be a comma here... the two lines are separate thoughts

```suggestion
Big companies start adopting Node.js: LinkedIn, Uber
```

Reply:
I agree with you. Is the intent to have 2 separate lines of text in these sections? This is what I see currently:
![image](https://user-images.githubusercontent.com/636987/53109381-c1434d00-3506-11e9-94da-d6d6509d09a3.png)
And that's where the motivation to add the comma comes in :grin: